test/test_orthonormal_basis.py

```diff
@@ -98,26 +98,30 @@ def test_prism(order):
         for j in range(ndofs):
             mat[i, j] = sum(basis[:, i] * basis[:, j] * Qwts)
 
-    np.set_printoptions(suppress=True)
-    print(mat)
+    # np.set_printoptions(suppress=True)
+    # print(mat)
     assert(np.isclose(mat * 8.0, np.eye(mat.shape[0])).all())
 
 
-@pytest.mark.parametrize("cell_type", [basix.CellType.interval,
-                                       basix.CellType.triangle,
-                                       basix.CellType.tetrahedron])
-@pytest.mark.parametrize("order", [0, 1, 2, 3, 4])
+@pytest.mark.parametrize("cell_type", [
+    basix.CellType.interval,
+    basix.CellType.triangle,
+    basix.CellType.tetrahedron
+])
+# @pytest.mark.parametrize("order", [0, 1, 2, 3, 4])
+@pytest.mark.parametrize("order", [1])
 def test_cell(cell_type, order):
     Qpts, Qwts = basix.make_quadrature("default", cell_type, 2*order + 1)
```

Comment:
Missing space around `*`. Shouldn't the flake8 checks pick this up? Could be a sign that flake8 isn't running everywhere?